trunk/source/Library-UltimateModel/src/de/uni_freiburg/informatik/ultimate/core/model/models/ILocation.java

```diff
@@ -67,6 +67,11 @@ public interface ILocation extends IAnnotations {
 	 */
 	int getEndColumn();
 
+	/**
+	 * @return The function that contains this {@code Location} (i.e. its scope), null if unknown.

```

Comment:
Oh, I thought null indicates the global scope. What distinguishes unknown from global scope?

Reply:
Good point! Maybe I can just say "null if unknown or not present" (or something similar) in the docs?